packages/nimble-components/src/anchor-menu-item/index.ts

```diff
@@ -67,21 +67,6 @@ export class AnchorMenuItem extends AnchorBase {
         return true;
     }
 
-    // The FAST Menu manages the `tabindex` of its menu items. Because of a bug in Chromium
```

Comment:
There is a workaround cited at the bottom of this file linking to a FAST PR that was closed: https://github.com/microsoft/fast/pull/6667

Can we remove that workaround?

Reply:
That PR was closed without merging, I think, because I unintentionally deleted the source branch. I'll resurrect the PR and push that forward, but I don't think we should keep this PR lingering around open until that goes in.